### PR TITLE
Feat/v1

### PR DESCRIPTION
**Pull request by polomani**
this is just for me and @mrtraser to keep track of the process
eventually this branch might become a default one

### CODE REVIEW

File: src/variable_query_ctrl.tsx [hunk 1]
```diff
@@ -1,31 +1,68 @@
...
+    to match defined prop and matched value will be included
+    <br />
+    <code className="query-keyword">'!~'</code> – regex inequality, means that provided regexp is
+    used to match defined prop and matched value will be excluded
+    <br />
+    <code className="query-keyword">'!='</code> – strict inequality, means that provided string is
+    used to strict prop comparing and matched value will be excluded
+    <br />
+    Example:{' '}
+    <code className="query-keyword">{`assets{metadata={KEY='value', KEY_2=~'value.*'} assetSubtreeIds=[{id=123}]}`}</code>
```
Comment (maciejmrowiec) [inline src/variable_query_ctrl.tsx:33]:
```suggestion
    <code className="query-keyword">{`assets{metadata={KEY='value', KEY_2=~'value.*'}, assetSubtreeIds=[{id=123}]}`}</code>
```

**Comment by maciejmrowiec** [comment]
Notes:
* Needs change log update (copy over from the docs site)
* README needs update -- move all usage/getting started and pointing the main grafana to docs.cognite.com (eliminate multiple docs covering this in various degrees)

File: src/partials/query.editor.html [hunk 2]
```diff
@@ -74,121 +76,108 @@
...
               ↪ yields the range of the timeseries aggregated by averages
-            <code class="query-keyword">power(timeseries{} - AVG(timeseries{}), 2)</code>
+            <code class="query-keyword">pow(ts{} - avg(ts{}), 2)</code>
               ↪ yields the squared deviation of each timeseries from the average <br>
-  If you want to reference a specific timeseries (for example, a timeseries that is not associated with an asset), use `[ID]`, `[ID,aggregate]`, or `[ID,aggregate,granularity]`.
-  Example:  <code class="query-keyword">SUM(timeseries{metadata.type="TEMP"}) - [123456,avg,12h]</code>
-      </pre>
+  There is a support for some advanced functions, like <code class="query-keyword">round</code>, <code class="query-keyword">on_error</code> and <code class="query-keyword">map</code>.
+  The documentation on how to use them can be found on <a class="query-keyword" href="https://docs.cognite.com/api">docs.cognite.com/api</a>
+    </pre>
```
Comment (maciejmrowiec) [inline src/partials/query.editor.html:181]:
```suggestion
  The documentation on how to use them can be found on <a class="query-keyword" href="https://docs.cognite.com/api">docs.cognite.com/api</a>
  
To learn more about the querying capabilities of Cognite Data Source for Grafana, please visit our <a class="query-keyword" href="https://docs.cognite.com/cdf/dashboards/guides/grafana/getting_started.html">documentation</a>.
    </pre>
```

Comment (maciejmrowiec) [reply src/partials/query.editor.html:181]: @polomani

File: src/partials/annotations.editor.html [hunk 3]
```diff
@@ -17,40 +17,23 @@
...
+  Example: <code class="query-keyword">events{type='WORKORDER', assetSubtreeIds=[{id=12}, {externalId='external'}]}</code>
+  Results filtering is available by adding <code class="query-keyword">'=~'</code>, <code class="query-keyword">'!~'</code> and <code class="query-keyword">'!='</code> signs to filtered props. Applying few filters for query acts as logic AND
+  Format:
+    <code class="query-keyword">'=~'</code> – regex equality, means that provided regexp is used to match defined prop and matched value will be included
+    <code class="query-keyword">'!~'</code> – regex inequality, means that provided regexp is used to match defined prop and matched value will be excluded
+    <code class="query-keyword">'!='</code> – strict inequality, means that provided string is used to strict prop comparing and matched value will be excluded
+  Example: <code class="query-keyword">events{type='WORKORDER', subtype=~'SUB.*'}</code>
+Variables templating is also available via <code class="query-keyword">$variable</code>
+  Example: <code class="query-keyword">events{type='WORKORDER', subtype=$variable}</code>
+      </pre>
```
Comment (maciejmrowiec) [inline src/partials/annotations.editor.html:36]:
```suggestion
  Example: <code class="query-keyword">events{type='WORKORDER', subtype=$variable}</code>
  
 To learn more about the querying capabilities of Cognite Data Source for Grafana, please visit our <a class="query-keyword" href="https://docs.cognite.com/cdf/dashboards/guides/grafana/getting_started.html">documentation</a>.
      </pre>
```

Comment (maciejmrowiec) [reply src/partials/annotations.editor.html:36]: @polomani

File: src/variable_query_ctrl.tsx [hunk 4]
```diff
@@ -1,31 +1,75 @@
...
+    <br />
+    <code className="query-keyword">'!~'</code> – regex inequality, means that provided regexp is
+    used to match defined prop and matched value will be excluded
+    <br />
+    <code className="query-keyword">'!='</code> – strict inequality, means that provided string is
+    used to strict prop comparing and matched value will be excluded
+    <br />
+    Example:{' '}
+    <code className="query-keyword">{`assets{metadata={KEY='value', KEY_2=~'value.*'}, assetSubtreeIds=[{id=123}]}`}</code>
+  </pre>
```
Comment (maciejmrowiec) [inline src/variable_query_ctrl.tsx:41]:
```suggestion
    <code className="query-keyword">{`assets{metadata={KEY='value', KEY_2=~'value.*'}, assetSubtreeIds=[{id=123}]}`}</code>
    
    To learn more about the querying capabilities of Cognite Data Source for Grafana, please visit our <a class="query-keyword" href="https://docs.cognite.com/cdf/dashboards/guides/grafana/getting_started.html">documentation</a>.
  </pre>
```

Comment (maciejmrowiec) [reply src/variable_query_ctrl.tsx:41]: @polomani

**Comment by maciejmrowiec** [comment]
Outside of the above code suggestions. The content has been updated for the release.
I moved all usage/install docs to docs portal, this is common practice to so it should not be surprising or look off. There are several advantages to doing so, in our case mainly: centralized content means maintainability, gives a more professional appearance, connects the connector to the broader CDF ecosystem, and teaches CDF users to depend on docs portal for accurate and up to date information.

**Comment by polomani** [comment]
closing in favour of https://github.com/cognitedata/cognite-grafana-datasource/pull/116

**Pull request closed without merging**